### ISSUE TITLE
Cannot update to 0.21.0

### ISSUE THREAD

**Issue by shaunlebron** [atom/language-clojure#44]
If I search for packages to install, I see `0.21.0`:

<img width="783" alt="screen shot 2016-06-05 at 1 56 24 pm" src="https://cloud.githubusercontent.com/assets/116838/15807518/d0ed660c-2b25-11e6-849d-a99688e26002.png">

I have the older `0.20.0`, but it is not detecting the newer version.  There is no update button:

<img width="757" alt="screen shot 2016-06-05 at 1 57 05 pm" src="https://cloud.githubusercontent.com/assets/116838/15807517/d0ed1382-2b25-11e6-8128-233958ed9f17.png">

Running `apm upgrade` didn't help either:

```
$ apm upgrade language-clojure
Package Updates Available (0)
└── (empty)
```

**Comment by shaunlebron** [comment]
Installing it like this seems work, but it's installing it a user package instead of upgrading the core package:

```
$ apm install language-clojure@0.21.0
The language-clojure package is bundled with Atom and should not be explicitly installed.
You can run `apm uninstall language-clojure` to uninstall it and then the version bundled
with Atom will be used.
Installing language-clojure@0.21.0 to /Users/swilliam/.atom/packages ✓
```

**Comment by 50Wliu** [comment]
This is expected, as core packages are associated with the Atom version they come with and cannot be manually updated (to prevent compatibility conflicts).  v0.21.0 should be available starting in Atom 1.9.0.

**Comment by shaunlebron** [comment]
Thanks for the background
